### PR TITLE
Fix manual uncle reward calculation

### PR DESCRIPTION
**Pull request by sl1depengwyn**
## Changelog

Take into account the case when there are two uncles in a single block

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: apps/explorer/lib/explorer/chain/wei.ex [hunk 1]
```diff
@@ -196,6 +197,22 @@ defmodule Explorer.Chain.Wei do
...
+  ## Example
+
+      iex> wei = %Explorer.Chain.Wei{value: Decimal.new(10)}
+      iex> divisor = 5
+      iex> Explorer.Chain.Wei.div(wei, divisor)
+      %Explorer.Chain.Wei{value: Decimal.new(2)}
+  """
+  def div(%Wei{value: value}, divisor) when Decimal.is_decimal(divisor) or is_integer(divisor) do
+    value
+    |> Decimal.div(divisor)
```
Comment (vbaranov) [inline apps/explorer/lib/explorer/chain/wei.ex:212]: I suppose, we should add divisor > 0 to the guard.

**Review by Qwerty5Uiop — APPROVED** [review]

File: apps/explorer/lib/explorer/chain/wei.ex [hunk 2]
```diff
@@ -196,6 +197,28 @@ defmodule Explorer.Chain.Wei do
...
+    |> from(:wei)
+  end
+
+  def div(%Wei{value: value}, %Decimal{sign: 1} = divisor) do
+    value
+    |> Decimal.div(divisor)
+    |> from(:wei)
+  end
```
Comment (vbaranov) [inline apps/explorer/lib/explorer/chain/wei.ex:220]: @sl1depengwyn should we add a fallback function clause and return `Decimal.new(0)` in all other cases?

File: apps/explorer/lib/explorer/chain/wei.ex [hunk 3]
```diff
@@ -196,6 +197,28 @@ defmodule Explorer.Chain.Wei do
...
+  Divides Wei values by an `t:integer/0` or `t:Decimal.t/0`.
+
+  ## Example
+
+      iex> wei = %Explorer.Chain.Wei{value: Decimal.new(10)}
+      iex> divisor = 5
+      iex> Explorer.Chain.Wei.div(wei, divisor)
+      %Explorer.Chain.Wei{value: Decimal.new(2)}
+  """
+  def div(%Wei{value: value}, divisor) when is_integer(divisor) and divisor > 0 do
```
Comment (nikitosing) [inline apps/explorer/lib/explorer/chain/wei.ex:210]: Please add spec

Comment (sl1depengwyn) [reply apps/explorer/lib/explorer/chain/wei.ex:211]: Done, thanks

**Review by nikitosing — APPROVED** [review]

**Pull request merged**